packages/cli/src/WorkflowRunner.ts

```diff
@@ -150,27 +153,27 @@ export class WorkflowRunner {
 		data: IWorkflowExecutionDataProcess,
 		loadStaticData?: boolean,
 		realtime?: boolean,
-		executionId?: string,
+		restartExecutionId?: string,
 		responsePromise?: IDeferredPromise<IExecuteResponsePromiseData>,
 	): Promise<string> {
-		await initErrorHandling();
+		const workflowId = data.workflowData.id;
+		if (loadStaticData === true && workflowId) {
+			data.workflowData.staticData =
+				await this.workflowStaticDataService.getStaticDataById(workflowId);
+		}
```

Comment:
Should static data not be loaded before we create the execution? In the current implementation when we load it in the worker in the queue mode, the static data isn't necessarily going to be a snapshot of the data at the correct time, since an execution could end up waiting for minutes when the worker is under load.
However, if static data should be loaded just before the execution actually starts, then we still need to make the implementation consistent between the two modes, since we'll have concurrency limits in `main` mode as well soon.